lib/canvas_test_helpers.dart

```diff
@@ -0,0 +1,7 @@
+library canvas_test_helpers;
+
+/// A Calculator.
+class Calculator {
+  /// Returns [value] plus 1.
+  int addOne(int value) => value + 1;
```

Comment:
We can remove this file

Reply:
Ah, I was going to put the exports in here, forgot about that.